modules/samples/sample-clients/tcp-server/src/main/java/org/wso2/sp/tcp/server/TCPServer.java

```diff
@@ -80,8 +80,17 @@ public void onEvent(Event event) {
         });
 
         ServerConfig serverConfig = new ServerConfig();
-        serverConfig.setHost("localhost");
-        serverConfig.setPort(Integer.parseInt("9893"));
+        String host = "localhost";
+        String port = "9893";
+        if (args[0] != null && !args[0].equals("")) {
```

Comment:
you can use StringUtils.isBlank()

Reply:
Usually, all the siddhi related repositories do not have org.apache.commons.lang3 dependency. Hence followed this way.